guides/common/modules/proc_installing-packages-on-a-host.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Installing_Packages_on_a_Host_{context}"]
+= Installing Packages on a Host
+
+Use this procedure to review and install packages on a host.
+The list of packages available for installation depends on the Content View and Lifecycle Environment assigned to the host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host you want to install packages on.
+. Navigate to the *Content* tab > *Packages* subtab.
```

Comment:
Unsure about using "navigate" again (which we generally don't) and "subtab" which I've only found two other instances in Foreman documentation. Maybe reword?